src/esm_runscripts/slurm.py

```diff
@@ -200,7 +200,17 @@ def het_par_headers(config, cluster, headers):
             # and partitions)
             for model in config["general"]["valid_model_names"]:
                 nodes = config[model].get("nodes")
-                if nodes:
+
+# kh 24.06.22 observed behavior was:
+#model:  echam, nodes:  4
+#model:  fesom, nodes:  32
+#model:  oasis3mct, nodes:  None
+#model:  recom, nodes:  None
+#model:  jsbach, nodes:  None
+#model:  hdmodel, nodes:  1
+
+# kh 24.06.22 workaround: filter hdmodel
+                if nodes and (model != "hdmodel"):
```

Comment:
```suggestion
                if nodes:
```
This should work if the previous change is taken